src/popup/components/TransactionLabel.vue

```diff
@@ -203,9 +200,11 @@ export default defineComponent({
         };
       }
 
-      const translation = transactionListTypes[txType.value!] || transactionTypes[txType.value!];
+      const translation = transactionListTypes[
+        camelCase(Tag[outerTxType.value!])] || transactionTypes[camelCase(Tag[outerTxType.value!])
+      ];
```

Comment:
I'd suggest better formatting here 
```suggestion
     const selectedType = camelCase(Tag[outerTxType.value!])
      const translation = transactionListTypes[selectedType] || transactionTypes[selectedType]
```